### PR TITLE
make urls relative

### PR DESCRIPTION
**Pull request by KillariDev**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
       %PUBLIC_URL% will be replaced with the URL of the `public` folder during build.
       Only files inside the `public` folder can be referenced from the HTML.
     -->
-    <link rel="shortcut icon" type="image/png" href="%PUBLIC_URL%/favicon.png" />
-    <link rel="apple-touch-icon" sizes="192x192" href="%PUBLIC_URL%/images/192x192_App_Icon.png" />
-    <link rel="apple-touch-icon" sizes="512x512" href="%PUBLIC_URL%/images/512x512_App_Icon.png" />
+    <link rel="shortcut icon" type="image/png" href="/favicon.png" />
+    <link rel="apple-touch-icon" sizes="192x192" href="/images/192x192_App_Icon.png" />
+    <link rel="apple-touch-icon" sizes="512x512" href="/images/512x512_App_Icon.png" />
```
Comment (MicahZoltu) [inline public/index.html:14]: Is leading `/` here correct, or should it be `./images...` or `images...`?  All 3 mean different things in HTML and we should make sure we are using the right one.  I think leading `/` means "relative to the host", which may not be correct.  I think what we want is "relative to index.html", but I forget how to express that.

Comment (MicahZoltu) [reply public/index.html:14]: Same comment for fonts below.

**Review by MicahZoltu — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (KillariDev) [reply public/index.html:14]: I imagined / is relative to the current folder you are in, which is the same folder where the index.html is

Comment (MicahZoltu) [reply public/index.html:14]: I recommend against imagining such things in HTML.  😁  I think MDN has a document that describes the different relative/absolute pathing options and how they work in HTML files.

**Pull request closed without merging**